resources/codegen_inputs/kern.rs

```diff
@@ -0,0 +1,94 @@
+#![parse_module(read_fonts::tables::kern)]
+
+/// The [kern (Kerning)](https://docs.microsoft.com/en-us/typography/opentype/spec/kern) table
+#[tag = "kern"]
+table Kern {
+    /// Table version number — set to 0.
+    #[compile(0)]
+    version: u16,
+    /// Number of subtables in the kerning table
+    #[compile(array_len($subtables))]
+    num_tables: u16,
+    #[count($num_tables)]
+    #[traverse_with(skip)]
+    subtables: VarLenArray<Kern0>,
+}
+
+///// The different kern subtable formats.
+//format u16 KernSubtable {
+    //Format0(Kern0),
+    //// Nope.
```

Comment:
lol ... explain why? Even just note we're starting simple?

Reply:
Just noticed https://github.com/googlefonts/fontations/issues/1183#issuecomment-2421870991 has some good reasons, "Windows only supports format 0, and FontTools does not know of it either. FontTools also mentions a "new" Apple-specific format  kern table (not subtable) with version 1.0 and completely different structures." ... should we mention 1 as well?